postgresql/hooks/init

```diff
@@ -10,10 +10,12 @@ chown -R hab:hab {{pkg.svc_var_path}}
 
 if [[ ! -f "{{pkg.svc_data_path}}/PG_VERSION" ]]; then
   echo " Database does not exist, creating with 'initdb'"
-  initdb -U {{cfg.initdb_superuser_name}} \
-         -E {{cfg.initdb_encoding}} \
+  openssl rand -base64 32 > {{pkg.svc_config_path}}/pwfile
```

Comment:
@bodymindarts good question. Our thinking here is that we need to implement "actions" as we chatted about. We'll use that from an app server binding to a DB server to ask for them to create a DB for them.

As is, this isn't super useful unless you set the password to an env var which is highly recommended against. You basically need to write a user.toml with this to make sure you have a password for the master db.

I made this change in preparation for that